### DISCUSSION TITLE
VITS 2 is there, will it be implemented?

### DISCUSSION THREAD

**Discussion by trunglebka** [rhasspy/piper#158] [Ideas]
They just released VITS 2 paper: https://arxiv.org/pdf/2307.16430.pdf
Piper by now is pretty fast but it may even faster :rocket:  if VITS 2 is used.

**Comment by rmcpantoja** [comment]
It would be nice, but the training and inference code isn't released yet.

**Reply by trunglebka** [reply]
Yeah, just want to inform piper maintainers that there is an update of Vits

**Reply by Feanix-Fyre** [reply]
I saw a Pytorch implementation of vits2 today with inference and training code available. Can't remember if it was official or unofficial.

**Reply by synesthesiam** [reply]
An important first step in getting VITS2 support in Piper is getting the model exported to onnx.

**Reply by Feanix-Fyre** [reply]
Found it. And it supports onnx export. I think there is only one public pretrained model but others are being worked on. 

https://github.com/p0p4k/vits2_pytorch
